requirements/prod.txt

```diff
@@ -1,13 +1,14 @@
-Authlib==1.3.1
-asgi-correlation-id==4.3.1
-SQLAlchemy[asyncio]==2.0.31
-alembic==1.13.2
 aiosqlite==0.20.0
-fastapi==0.111.1
-pydantic==2.8.2
-pydantic-settings==2.3.4
-python-dotenv==1.0.1
-httpx==0.27.0
-uvicorn==0.30.3
+alembic==1.14.0
+asgi-correlation-id==4.3.4
+Authlib==1.3.2
+fastapi==0.115.6
+httpx==0.27.2
```

Comment:
Can't update to httpx 28.1 because of https://github.com/lundberg/respx/issues/277